e2e-tests/README.md

```diff
@@ -4,23 +4,42 @@
 npx playwright install
 ```
 
-The instructions below assume Headlamp is running in-cluster with minikube.
+The instructions below assume Headlamp is running locally in-cluster with minikube.
```

Comment:
I think this is implicit and less confusing to leave as it was.

Reply:
done!